src/orderbook.js

```diff
@@ -1197,6 +1334,37 @@ OrderBook.prototype.computeAutobridgedOffers = function() {
   this._offersAutobridged = autobridgeCalculator.calculate();
 };
 
+OrderBook.prototype.computeAutobridgedOffersWrapper = function() {
+  var startTime = Date.now();
+  this.computeAutobridgedOffers();
+  this.mergeDirectAndAutobridgedBooks();
+  var lasted = (Date.now() - startTime) * 5;
+  let m = this._autobridgeThrottleTimeMultiplier;
+  while (lasted > m * OrderBook.AUTOBRIDGE_CALCULATE_THROTTLE_TIME) {
+    ++m;
```

Comment:
I don't think we needed that. I think autobridged book size will be not change much during user session, so time needed to process it will be constant too. And if user device is slow, it will not get faster.
This code was written with mobile devices in mind.